chainerui/tasks/crawl_result.py

```diff
@@ -46,14 +46,32 @@ def crawl_result_path(result_path):
     return result
 
 
-def crawl_result(result_id, force=None):
+def _check_log_updated(result):
+    log_json_path = os.path.join(result.path_name, 'log')
+    if not os.path.isfile(log_json_path):
+        # log file is removed, so don't have to update
+        return False
+
+    current_modified_at = result.log_modified_at
+    modified_at = datetime.datetime.fromtimestamp(
+        os.path.getmtime(log_json_path))
+    if current_modified_at is None or current_modified_at != modified_at:
+        result.log_modified_at = modified_at
+        return True
+
+    return False
+
+
+def crawl_result(result_id, force=False):
     """crawl_results."""
 
     current_result = DB_SESSION.query(Result).filter_by(id=result_id).first()
 
     now = datetime.datetime.now()
 
-    if force is None and (now - current_result.updated_at).total_seconds() < 4:
+    if (not force) and (now - current_result.updated_at).total_seconds() < 4:
+        return current_result
+    if (not force) and not _check_log_updated(current_result):
```

Comment:
This condition does not allow to get new commands and snapshot history. Even if log file is not modified, result model should be updated commands and snapshot information.